arch/x86/cpu/tangier/pinmux.c

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
Yep, like I said in the bug report, check _pinctrl-sandbox.c_ for a hint how this should look like. (Yes, I understand that we are creating a limited driver, it needs to be taken into account)

Reply:
Okay. So we're making it the UCLASS_PINCTRL, then? With some of pinctrl_ops implemented. And then we'll configure the pins in edison.c using the pinctrl API? Is this the plan? If so, I'll need some time adapting this code to the pinctrl API.

Reply:
I'm pretty sure that this what upstream ask us during review. However, we can try first with reduced solution. I guess, we may send it upstream and see what they will tell. In any case it would be nice to have your stuff in my tree sooner.